scripts/README.md

```diff
@@ -177,4 +178,28 @@ The output is following format.
   {2} : The column number that a control-code exists.
   {3} : The found control-code.
   {4} : The one-line strings in the file.
+```
+
+## draft-translate.py
+
+This script creates a draft translation file for a given file and language.
+The output draft translation file is created respect to the given file path and the given language.
+(ex. ../content/en/docs/concepts/storage/storage-limits.md => ../content/ko/docs/concepts/storage/storage-limits.md)
```

Comment:
```suggestion
(For example: ../content/en/docs/concepts/storage/storage-limits.md => ../content/ko/docs/concepts/storage/storage-limits.md)
```
Avoid Latin phrases,
Prefer English terms over Latin abbreviations.
https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases

Reply:
For the README, it's less important to avoid Latin phrases. No problem to use “For example” though.

Reply:
@ydFu 
`For example` would be better, thank you